src/04_printing.py

```diff
@@ -12,6 +12,17 @@
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
 
+# % is the string modulo operator read more about it: https://www.python-course.eu/python3_formatted_output.php
+
+print("The value of %(x)s is ten, the value of %(y)s is a decimal, and the value of %(z)s is <3 turtles."
```

Comment:
print("The value of %(x)s is ten, the value of %(y)s is a decimal, and the value of %(z)s is <3 turtles.")

Reply:
I'm not sure I understand what you mean